packages/gdb.rb

```diff
@@ -3,28 +3,40 @@
 class Gdb < Package
   description 'GDB, the GNU Project debugger, allows you to see what is going on \'inside\' another program while it executes -- or what another program was doing at the moment it crashed.'
   homepage 'https://www.gnu.org/software/gdb/'
-  version '8.1'
-  source_url 'https://ftpmirror.gnu.org/gdb/gdb-8.1.tar.xz'
-  source_sha256 'af61a0263858e69c5dce51eab26662ff3d2ad9aa68da9583e8143b5426be4b34'
+  version '8.2.1'
+  source_url 'https://ftpmirror.gnu.org/gdb/gdb-8.2.1.tar.xz'
+  source_sha256 '0a6a432907a03c5c8eaad3c3cffd50c00a40c3a5e3c4039440624bae703f2202'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gdb-8.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gdb-8.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gdb-8.1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gdb-8.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '0487578f13f93e89ad87ec45a1468e057055465a83de7ee8a1cd27a4b467732b',
-     armv7l: '0487578f13f93e89ad87ec45a1468e057055465a83de7ee8a1cd27a4b467732b',
-       i686: 'e4078ff2566028ca81aa00ee3b47bfc4723fa8d75422fc80b463e8273844aef0',
-     x86_64: '8e7f25bb702e9bdf3b1d657aee7cf79e6de0cfdcf6bb512159818c02e39e2593',
   })
 
-  depends_on "ncurses"
-  depends_on "texinfo"
+  depends_on 'texinfo'
+  depends_on 'libx11'
+  depends_on 'expat'
+  depends_on 'python3'
+  depends_on 'python27'
+  depends_on 'ncurses'
+  depends_on 'readline7'
 
   def self.build
-    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
+    system './configure',
+           '--with-x',
+           '--with-lzma',
+           '--enable-sim',
+           '--with-expat',
+           '--enable-tui',
+           '--with-python',
+           '--with-curses',
+           '--enable-shared',
+           '--with-system-zlib',
+           '--enable-64-bit-bfd',
```

Comment:
Isn't this for x86_64 systems only?  What about arm and i686?

Reply:
Wouldn't it be better to leave out the option if it's enabled by default?  Otherwise, you are explicitly enabling it on non-64-bit systems.

Reply:
The option is useful for cross-compiling to target a 64-bit system, but has other uses too (among them, one is performance).

Reply:
We're not cross-compiling so there is no need to set this option.

Reply:
We may in the future. I don't want to close that door off for other people. As I said before, this option was introduced for compatibility with libbfd from binutils, as two versions of libbfd compiled with different options cause... problems.

Reply:
It causes no harm, so there is no reason to remove it.

Reply:
You have never attempted to compile on other systems so how can you be so sure?